system-tests/resources/webdid/gaia-x/did.json

```diff
@@ -0,0 +1,29 @@
+{
+	"id": "did:web:did-server:gaiax",
+	"@context": [
+		"https://www.w3.org/ns/did/v1",
+		{
+			"@base": "did:web:did-server:gaiax"
+		}
+	],
+	"service": [{
+		"id": "#identity-hub-url",
+		"type": "IdentityHub",
+		"serviceEndpoint": "http://dummy?region=eu"
+	}],
```

Comment:
nit: do we also need IdentityHub entry in this did for gaiax as well ?

Reply:
We don't need it for the GAIA-X authority. There is not even an Identity Hub running for it.

Reply:
removed